cli/ssh_client.go

```diff
@@ -16,6 +17,10 @@ import (
 type sshClient struct {
 	manager jasper.Manager
 	opts    sshClientOptions
+	seCache struct {
```

Comment:
Should the usage of `se` be changed to `sh`? This is applied inconsistently throughout the code.